lib/config/rules/react.js

```diff
@@ -47,6 +49,8 @@ module.exports = {
   'react/prop-types': 'warn',
   // Prevent missing React when using JSX
   'react/react-in-jsx-scope': 'error',
+  // Enforce a defaultProps definition for every prop that is not a required prop
+  'react/require-default-props': 'error',
```

Comment:
IMO this rule is not worth it, I think unset values are fine for boolean props (which would default to falsey values anyways), and I kind of prefer specifying defaults in actual JS rather than `defaultProps`.